src/services/partner.ts

```diff
@@ -0,0 +1,56 @@
+import { CallOverrides } from "@ethersproject/contracts";
+
+import { ChainId } from "../chain";
+import { ContractService } from "../common";
+import { Context } from "../context";
+
+const YVECRV = "0xc5bDdf9843308380375a611c18B50Fb9341f502A";
+const PICKLE = "0xCeD67a187b923F0E5ebcc77C7f2F7da20099e378";
```

Comment:
we named this one as `PSLPYVBOOSTETH` in frontend, might be good to be more explicit and in sync with that